scripts/v.db.univar/v.db.univar.html

```diff
@@ -66,11 +66,7 @@ <h2>SEE ALSO</h2>
 <a href="v.random.html">v.random</a>
 </em>
 
-<h2>AUTHOR</h2>
- Michael Barton, Arizona State University
-<p>and authors of <em>r.univar.sh</em> (Markus Neteler et al.)
+<h2>AUTHORS</h2>
 
-<!--
-<p>
-<i>Last changed: $Date$</i>
--->
+Michael Barton, Arizona State University
+<p>and authors of <em>r.univar.sh</em> (Markus Neteler et al.)
```

Comment:
Line 43 unnecessary one empty line.

Reply:
Fixed as for `scripts/v.db.addcolumn/v.db.addcolumn.html` (see above)